src/frontend/src/expr/now.rs

```diff
@@ -34,9 +34,11 @@ impl Expr for Now {
     }
 
     fn to_expr_proto(&self) -> ExprNode {
-        unreachable!(
-            "`Now` should be translated to `Literal` in batch mode or `NowNode` in stream mode"
-        )
+        ExprNode {
+            expr_type: expr_node::Type::Now.into(),
```

Comment:
...so is it possible to simply fill `Unspecified` here? However we should carefully review all current usages of `get_expr_type().unwrap()` and replace them with `expr_type()`.